.eslintrc.json

```diff
@@ -28,7 +28,8 @@
       {
         "ignore": [
           0,
-          1
+          1,
+          2
```

Comment:
Please remove `2`. Use constants to fix ESLint issue: https://eslint.org/docs/rules/no-magic-numbers